waspc/src/Wasp/JsImport.hs

```diff
@@ -13,7 +13,9 @@ data JsImport = JsImport
     --   So for example if jsImportFrom is "test.js", we expect file
     --   to exist at <external_code_dir>/test.js.
     --   TODO: Make this more explicit in the code (both here and in wasp lang)? Also, support importing npm packages?
-    , jsImportFrom :: !Path.RelFile
+    -- TODO: Use StrongPath here? Be relative to source external code or generated external code?
+    --   Maybe don't be relative to any of them and drop this assumption that "from" is relative to the ext code?
+    , jsImportFrom :: !(P.Path P.Rel P.File)
```

Comment:
I will soon be dealing with JS import as part of queries/actions task, so I thought I would leave this for then, when I will better understand how I want jsImportFrom to behave like.